client/my-sites/exporter/exporter.jsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import Gridicon from 'components/gridicon';
+import Button from 'components/forms/form-button';
+import AdvancedOptions from 'my-sites/exporter/advanced-options';
+
+export default React.createClass( {
+	displayName: 'Exporter',
+
+	render: function() {
+		return (
+			<div className="section-export">
+				<CompactCard>
+					<header>
+						<Button
+							className="exporter__export-button"
+							disabled={ false }
+							isPrimary={ true }
+						>
+							{ this.translate( 'Export' ) }
+						</Button>
+						<h1 className="exporter__title">
+							{ this.translate( 'Download an Export File' ) }
+						</h1>
+					</header>
+					<a href="#" onClick={ this.props.toggleAdvancedSettings }>
+						<Gridicon
+							icon={ this.props.advancedSettings.isVisible ? 'chevron-up' : 'chevron-down' }
+							size={ 16 } />
+						{ this.translate( 'Advanced Export Settings' ) }
+					</a>
+				</CompactCard>
+
+				{
+					this.props.advancedSettings.isVisible &&
+					<AdvancedOptions
```

Comment:
@nb @aduth Thank you for the reviews and discussion. This is something that confuses me about Redux - it seems to solve a lot of problems by introducing a central store, but on the other hand it's like introducing a major god object :smile: 

I think there's two different issues here that it might help to look at in different ways:
 - **Persistent/API Data:** For persistent data, the component hierarchy often won't match the data structure. For example, the post editor component tree doesn't match the `post` object tree (for good reason), which makes it difficult to tell which components are affecting what data. There you have the situation where multiple components at various nesting levels in completely different trees are manipulating an object at various nesting levels and causing issues like the unnecessary re-renders (ref: p4TIVU-11B-p2). I'm not totally familiar with Redux selectors but it seems like they could come in handy here - passing them down to components rather than the raw data itself could help in making data access more granular.
 - **UI state data** In the case of this PR, the data introduced here is purely for UI display purposes so the data hierarchy can be matched exactly with the component hierarchy. For example, the `AdvancedOptions` component only uses the immediate children of the `advancedSettings` object [1]. It then passes down the `posts`, `pages`, and `feedback` objects to corresponding `OptionFieldset` components, which in turn use only the direct children of that data structure. It seems like at this stage it would be overkill to de-normalize this data structure to avoid nesting of the data?

[1] Advanced Options and Settings are actually the same, somehow I ended up naming them two things - I'll change them both to 'Settings'.